lightbar_manager/src/lightbar_manager/lightbar_manager_sm.cpp

```diff
@@ -68,12 +68,12 @@ void LightBarManagerStateMachine::handleStateChange(const cav_msgs::GuidanceStat
     switch (msg_ptr->state)
     {
         case cav_msgs::GuidanceState::STARTUP:
-        case cav_msgs::GuidanceState::DRIVERS_READY:
         case cav_msgs::GuidanceState::SHUTDOWN:
-        case cav_msgs::GuidanceState::INACTIVE:
         onDisengage();
         break;
         case cav_msgs::GuidanceState::ACTIVE:
+        case cav_msgs::GuidanceState::DRIVERS_READY:
```

Comment:
I see no functional problem with these code changes.  However, readability is not good.  Please add a blank line after each break statement, and indent the code that is subordinate to each case statement, so it is really obvious which code is being executed by which case.